samples/meetings-live-code-interview/csharp/MeetingLiveCoding/ClientApp/package.json

```diff
@@ -38,10 +37,10 @@
   },
   "scripts": {
     "clean": "npx shx rm -rf build",
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
+    "build": "react-scripts --openssl-legacy-provider build",
```

Comment:
Why we have added these, share the RCA

Reply:
I built and tested my solution locally (http://localhost:3978/), but it does not work. So that is the reason I have added this RCA.

Reply:
Getting error related to npm start if we are not adding it.